lib/compat/wordpress-6.3/rest-api.php

```diff
@@ -65,7 +65,7 @@ function gutenberg_register_global_styles_revisions_endpoints() {
  * Registers the Global Styles REST API routes.
  */
 function gutenberg_register_global_styles_endpoints() {
-	$global_styles_controller = new Gutenberg_REST_Global_Styles_Controller_6_3();
+	$global_styles_controller = new Gutenberg_REST_Global_Styles_Controller();
```

Comment:
`gutenberg_register_global_styles_endpoints` should be moved to lib/experimental/rest-api.php

Reply:
I can do that quickly :-)